### ISSUE TITLE
Attentuation for Directional/Sun Lights when importing Blender files

### ISSUE THREAD

**Issue by amras0000** [mosra/magnum-plugins#120]
```c++
_importer = _manager.loadAndInstantiate("BlenderImporter");
_importer->openFile("lights.blend")
Containers::Optional<Trade::LightData> l = _importer->light(0);
```

Blender 3.0.0 allows for 4 types of light by default: Point, Sun, Spot, and Area. Here are the results when I run the importer on each of these lights:

```
Loading point.blend ...
Found a light of type Trade::LightData::Type::Point !
Loading spot.blend ...
Found a light of type Trade::LightData::Type::Spot !
Loading area.blend ...
Trade::AssimpImporter::light(): light type 5 is not supported
No light Found !
Loading sun.blend ...
Trade::LightData: attenuation has to be (1, 0, 0) for an ambient or directional light but got Vector(0, 0, 0)
Aborted (core dumped)
```

Area lights are not supported, and gracefully pass with a warning and a `NullOpt`. Directional (ie. sun) lights however, note a different attenuation notation and crash before `light()` exits - meaning the user has no opportunity to fix the attenuation to the expected value.

Test code and blender files are attached.

[magnum-blender-directional-light.zip](https://github.com/mosra/magnum-plugins/files/7999439/magnum-blender-directional-light.zip)

**Comment by pezcode** [comment]
Looks like this comment was too optimistic:
https://github.com/mosra/magnum-plugins/blob/4baaebe80fa08a72aeeca71b16e8e05ce16b57e0/src/MagnumPlugins/AssimpImporter/AssimpImporter.cpp#L895

That'll need hardcoded attentuation `{1, 0, 0}` for ambient and directional lights.

**Comment by mosra** [comment]
Should be fixed with 7936009a2c21a772051ddec4e562b51bb76520a4. If not, please complain :)
